gpm_api/io/filter.py

```diff
@@ -89,7 +89,10 @@ def _filter_filepath(filepath, product=None, version=None, start_time=None, end_
 
     """
 
-    info_dict = get_info_from_filepath(filepath)
+    try:
```

Comment:
The try - except was not for the "product" below ?

Reply:
The try - except was originally around [this line](https://github.com/EPFL-ENAC/gpm_api/blob/e002278de09db1aea9d826dc7f7a72e683126068/gpm_api/io/info.py#L119C28-L119C28). But since the `_get_info_from_filename` is emitting exceptions anyway, I thought it made sense to let `_get_info_from_filename` throw an exception when the product is not found, and catch it in `_filter_filepath` instead. What do you think @ghiggi?

Reply:
mmh ok with that. You can merge when you prefer